projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -314,13 +275,6 @@ public static function disable_forms_view_script_concat( $do_concat, $handle ) {
 	 * Register the contact form block.
 	 */
 	private static function register_contact_form_blocks() {
```

Comment:
I think it may be worth adding a comment here to explain that this registers the child blocks only, and that the parent block is registered separately so it can be more readily available?

Or maybe we could bring everything that's block-related to the new block class, to keep things together?